backend/auth-service/src/main/resources/application-local.yaml

```diff
@@ -0,0 +1,4 @@
+spring:
+  data:
+    mongodb:
+      uri: mongodb://mongo:mongo@localhost:27017/it-service-db?authSource=admin
```

Comment:
change the db to auth specific, like this: `auth`

Reply:
fixed